x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/operator/OperatorPrivileges.java

```diff
@@ -102,7 +102,14 @@ public void maybeMarkOperatorUser(Authentication authentication, ThreadContext t
             final User user = authentication.getEffectiveSubject().getUser();
             // Let internal users pass, they are exempt from marking and checking
```

Comment:
We should remove this comment as well.

Reply:
but isn't it still partially relevant?
I changed the first part to `Let internal users pass, and mark him as an operator`